### DISCUSSION TITLE
Any idea how to use evil key-bindings with eaf-terminal?

### DISCUSSION THREAD

**Discussion by XushanLu** [emacs-eaf/emacs-application-framework#1001] [Q&A] [answered]
Well, I guess not many people are using evil (under Spacemacs) while using EAF. But I am just doing that. 

Are there plans to support evil?

**Comment by manateelazycat** [comment, marked as answer]
https://github.com/emacs-eaf/emacs-application-framework/wiki/Evil

**Reply by XushanLu** [reply]
多谢！
